test/e2e/network/netpol/network_policy.go

```diff
@@ -1077,7 +1077,7 @@ var _ = common.SIGDescribe("Netpol", func() {
 
 			ginkgo.By("Trying to connect to TCP port 81, which should be blocked by the deny-ingress policy.")
 			reachability := NewReachability(model.AllPods(), true)
-			reachability.ExpectAllIngress(NewPodString(nsX, "a"), false)
+			reachability.ExpectPeer(&Peer{}, &Peer{Namespace: nsX}, false)
```

Comment:
What does this mean? It seems to me that the original assertion was "all connections to pod a in nsX should fail, but everything else should succeed" (which is correct), while the new assertion is "all connections from any pod in nsX to anywhere should fail, but everything else should succeed"... which is totally wrong. Or am I misunderstanding the reachability dsl?

Reply:
This says that ingress to all pods in nsX should be denied but pods from nsX can connect to pods in nsY or nsZ. 

In this test, the very first policy we have is to `deny ingress` to all pods(empty label selector) in nsX. We then apply a policy to allow SCTP traffic to pod a in nsX but this policy is OR'ed with the previous ones so ingress to other pods in nsX is still denied. 

Originally we were checking for only pod a in reachability matrix and the new assertion checks for all pods in nsX.

Reply:
doh, I mentally swapped the `from` and `to` when reading the definition of `ExpectPeer`. ok, so it doesn't say "all connections from any pod in nsX to anywhere should fail, but everything else should succeed", it says "all connections _to_ any pod in nsX from anywhere should fail, but everything else should succeed". OK, got it